definitions/features/foreman_1_7_x.rb

```diff
@@ -35,7 +35,10 @@ def custom_iptables_chain(name, rules)
 
   def del_custom_iptables_chain(name)
     return unless execute?("iptables -L #{name}") # the chain is already gone
-    execute!("iptables -D INPUT -j #{name}")
+    output =  execute("iptables -L INPUT | awk '$1 ~ /#{name}/ { print }'").strip
+    unless output.empty?
```

Comment:
what about using this as a condition: `execute?("iptables -L INPUT | tail -n +3 | grep '^#{name} '")`